sycl/test-e2e/KernelCompiler/kernel_compiler_sycl_jit_lifetimes.cpp

```diff
@@ -0,0 +1,104 @@
+//== kernel_compiler_sycl_jit_lifetimes.cpp - kernel_compiler lifetime tests
```

Comment:
looks like a bit wrong formatting, I believe the beginning and the ending of copyright note should have the same length.

Reply:
Filename was a tad too long, I renamed it and fixed formatting